TESTING.md

```diff
@@ -31,7 +31,7 @@ in another terminal
 ```bash
 cd integration
 npm install
-SERVER_URL=http://localhost:8080/query ./node_modules/.bin/graphql get-schema
+./node_modules/.bin/graphql-codegen
```

Comment:
use graphql-codegen because need to upgrade graphql-js to v16.3.0 (the same as gqlparser) and graphql-cli is wrapper for it